pkg/app/app_reconcile.go

```diff
@@ -103,6 +105,13 @@ func (a *App) reconcileDeploy() error {
 }
 
 func (a *App) reconcileFetchTemplateDeploy() exec.CmdRunResult {
+	reconcileStartTime := time.Now()
+	a.isFirstReconcile = a.appMetrics.ReconcileCountMetrics.GetReconcileAttemptCounterValue("app", a.app.Name, a.app.Namespace) == 1
+	defer func() {
+		a.appMetrics.ReconcileTimeMetrics.RegisterOverallTime(appResourceType, a.app.Name, a.app.Namespace, a.isFirstReconcile,
```

Comment:
nit: We can use `a.Name()` and `a.Namespace()` instead of `a.app.Name` or `a.app.Namespace`. 
Same for other places.